docs/sources/alerting/set-up/provision-alerting-resources/terraform-provisioning/index.md

```diff
@@ -11,41 +11,44 @@ keywords:
   - Terraform
 labels:
   products:
+    - cloud
     - enterprise
     - oss
-title: Use Terraform to manage alerting resources
+menuTitle: Use Terraform to provision
+title: Use Terraform to provision alerting resources
 weight: 200
 ---
 
-# Use Terraform to manage alerting resources
+# Use Terraform to provision alerting resources
 
 Use Terraform’s Grafana Provider to manage your alerting resources and provision them into your Grafana system. Terraform provider support for Grafana Alerting makes it easy to create, manage, and maintain your entire Grafana Alerting stack as code.
 
-For more information on managing your alerting resources using Terraform, refer to the [Grafana Provider](https://registry.terraform.io/providers/grafana/grafana/latest/docs) documentation.
+For more examples and information about the Terraform Alerting schemas, refer to the [Grafana Provider](https://registry.terraform.io/providers/grafana/grafana/latest/docs) documentation.
 
 Complete the following tasks to create and manage your alerting resources using Terraform.
 
 1. Create an API key for provisioning.
-1. Configure the Terraform provider.
-1. Define your alerting resources in Terraform.
-1. Run `terraform apply` to provision your alerting resources.
+2. Configure the Terraform provider.
+3. Define your alerting resources in Terraform. [Export alerting resources][alerting_export] in Terraform format, or implement the [Terraform Alerting schemas](https://registry.terraform.io/providers/grafana/grafana/latest/docs).
```

Comment:
```suggestion
1. Configure the Terraform provider.
1. Define your alerting resources in Terraform. [Export alerting resources][alerting_export] in Terraform format, or implement the [Terraform Alerting schemas](https://registry.terraform.io/providers/grafana/grafana/latest/docs).
```

Reply:
we use lazy numbering in markdown is always 1. 1. 1. and it renders correctly https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#ordered-lists